controllers/gitopsservice_controller.go

```diff
@@ -346,59 +333,8 @@ func (r *ReconcileGitopsService) reconcileDefaultArgoCDInstance(instance *pipeli
 		} else {
 			return reconcile.Result{}, err
 		}
-	} else {
```

Comment:
This changes are not required. There are added to ensure that during upgrade(v1.1 to v1.2 or above) all the workloads are created with resource requests and limits. 

Since we have decided to remove the resource quota for default Argo CD instance namespace, This code can be removed.